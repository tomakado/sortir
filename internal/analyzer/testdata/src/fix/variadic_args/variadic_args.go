package variadic_args

import "fmt"

func main() {
	fmt.Println("zebra", "apple", "banana") // want "variadic arguments are not sorted"
	
	fmt.Printf("%s %s %s\n", "z", "a", "b") // want "variadic arguments are not sorted"
}