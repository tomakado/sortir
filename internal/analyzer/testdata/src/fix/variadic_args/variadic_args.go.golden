package variadic_args

import "fmt"

func main() {
	fmt.Println("apple", "banana", "zebra") // want "variadic arguments are not sorted"
	
	fmt.Printf("%s %s %s\n", "a", "b", "z") // want "variadic arguments are not sorted"
}